agent/src/main/scala/za/co/absa/atum/agent/dispatcher/HttpDispatcher.scala

```diff
@@ -64,6 +65,16 @@ class HttpDispatcher(config: Config) extends Dispatcher with Logging {
     safeResponseBody(response).get
   }
 
+  override def saveAdditionalData(metadataDTO: AdditionalDataSubmitDTO): Unit = {
```

Comment:
```suggestion
  override def saveAdditionalData(additionalDataSubmitDTO: AdditionalDataSubmitDTO): Unit = {
```

and then its usage in the function